crates/bevy_render/src/camera/camera.rs

```diff
@@ -90,12 +90,12 @@ pub struct ComputedCameraValues {
 ///
 /// <https://en.wikipedia.org/wiki/Exposure_(photography)>
 #[derive(Component)]
-pub struct ExposureSettings {
+pub struct Exposure {
```

Comment:
This could use a mention in the migration guide

Reply:
ExposureSettings didn't exist in 0.12

Reply:
Right, oops.

Reply:
The migration guide of 11347 would need to be adjusted though, not sure how that's usually handled

Reply:
Good point